src/mixins/nft.js

```diff
@@ -248,7 +264,7 @@ export default {
           this.uiSetTxStatus(TX_STATUS.COMPLETED);
```

Comment:
Add another log here
```suggestion
          this.uiSetTxStatus(TX_STATUS.COMPLETED);
          logTrackerEvent(this, 'NFT', 'NFTCollectPurchaseCompleted'', this.classId, 1);
```